### PR TITLE
Add run task script

### PR DESCRIPTION
**Pull request by abtris**

### CODE REVIEW

**Comment by abtris** [comment]
It's looks that `test` command not work correctly.

File: docker/run.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+# Read DOMAIN and ORIG_SPF
+. $HOME/.spf-toolsrc
+echo "$DOMAIN - $ORIG_SPF"
+echo "$(date)"
+$HOME/spf-tools/compare.sh $DOMAIN $ORIG_SPF || despf.sh $ORIG_SPF \
```
Comment (nadade) [inline docker/run.sh]: @abtris I can see that the full path is only present when calling the first script; shouldn't you rather cd to this path before running all of them?

Comment (abtris) [inline docker/run.sh]: good point!

**Comment by abtris** [comment]
@nadade I change include ENV properties to use `docker run -e DOMAIN ...` and test it before run.

File: Dockerfile [hunk 2]
```diff
@@ -14,4 +14,4 @@ COPY . /home/spf-user/spf-tools
 
 USER spf-user
 
-CMD ["./spf-tools/runspftools.sh"]
\ No newline at end of file
+CMD ["./spf-tools/runspftools.sh"]
```
Comment (nadade) [inline Dockerfile]: run.sh instead of runspftools.sh?

Comment (abtris) [inline Dockerfile]: good point, I change this too.

**Comment by nadade** [comment]
@abtris :+1: good to go!

**Pull request merged**